### ISSUE TITLE
No binary for release 3.24.7

### ISSUE THREAD

**Issue by M-Whitaker** [mono0926/LicensePlist#215]
https://github.com/mono0926/LicensePlist/releases/tag/3.24.7 has no LicensePlistBinary-macos.artifactbundle.zip which is causing homebrew to fail:

```
==> Upgrading 1 outdated package:
mono0926/license-plist/license-plist 3.24.5 -> 3.24.7
==> Fetching mono0926/license-plist/license-plist
==> Downloading https://github.com/mono0926/LicensePlist/archive/3.24.7.tar.gz
Already downloaded: /Users/matwhi01/Library/Caches/Homebrew/downloads/6519503276f1a7085dc92e7b85030a43bd89dde39fea3c95e5bc3e47b1b3d9e3--LicensePlist-3.24.7.tar.gz
==> Upgrading mono0926/license-plist/license-plist
  3.24.5 -> 3.24.7
==> make install PREFIX=/opt/homebrew/Cellar/license-plist/3.24.7
Last 15 lines from /Users/matwhi01/Library/Logs/Homebrew/license-plist/01.make:
warning: 'swift-argument-parser': failed storing manifest for 'swift-argument-parser' in cache: attempt to write a readonly database
warning: 'swift-log': failed storing manifest for 'swift-log' in cache: attempt to write a readonly database
warning: 'loggerapi': failed storing manifest for 'loggerapi' in cache: attempt to write a readonly database
warning: 'flatten': failed storing manifest for 'flatten' in cache: attempt to write a readonly database
warning: 'shlist': failed storing manifest for 'shlist' in cache: attempt to write a readonly database
Downloading binary artifact https://github.com/mono0926/LicensePlist/releases/download/3.24.7/LicensePlistBinary-macos.artifactbundle.zip
2023-04-03 17:23:17.560 swift-build[12455:46617] NetworkStorageDB:_openDBReadConnections: failed to open read connection to DB @ /Users/matwhi01/Library/Caches/swift-build/Cache.db.  Error=14. Cause=unable to open database file
2023-04-03 17:23:17.560 swift-build[12455:46617] The read-connection to the DB=/Users/matwhi01/Library/Caches/swift-build/Cache.db is NOT valid.  Unable to determine schema version.
2023-04-03 17:23:17.560 swift-build[12455:46617] NetworkStorageDB:_openDBWriteConnections: failed to open write connection to DB @ /Users/matwhi01/Library/Caches/swift-build/Cache.db.  Error=14. Cause=unable to open database file
2023-04-03 17:23:17.560 swift-build[12455:46617] DEBUG: there is no SQL cache DB located at /Users/matwhi01/Library/Caches/swift-build/Cache.db.
2023-04-03 17:23:17.560 swift-build[12455:46617] DEBUG: there is no SQL cache DB located at /Users/matwhi01/Library/Caches/swift-build/Cache.db-shm.
2023-04-03 17:23:17.560 swift-build[12455:46617] DEBUG: there is no SQL cache DB located at /Users/matwhi01/Library/Caches/swift-build/Cache.db-wal.
[9/9] Downloading https://github.com/mono0926/LicensePlist/releases/download/3.24.7/LicensePlistBinary-macos.artifactbundle.zip
error: failed downloading 'https://github.com/mono0926/LicensePlist/releases/download/3.24.7/LicensePlistBinary-macos.artifactbundle.zip' which is required by binary target 'LicensePlistBinary': badResponseStatusCode(404)
make: *** [build] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/mono0926/homebrew-license-plist/issues
```

**Comment by p-linnane** [comment]
Also seen in https://github.com/Homebrew/homebrew-core/pull/127433

**Comment by mono0926** [comment]
@M-Whitaker @p-linnane 

Thanks for the report.
Fixed in v3.24.8 👌 
https://github.com/mono0926/LicensePlist/releases/tag/3.24.8
